sql/moz-fx-data-shared-prod/telemetry/active_users_aggregates/view.sql

```diff
@@ -3,6 +3,43 @@ CREATE OR REPLACE VIEW
 AS
 SELECT
   *,
-  `mozfun.norm.os`(os) AS os_grouped
+  dau AS qdau
```

Comment:
Not blocking, but is it possible to exclude the `qdau` column from the view?

Eventually, we have a goal to only have the concept of "dau" and not to have any concept of "qdau". Instead of specifying "qualified daily active users", the "qualified" part will be implicit in the definition of "active users". This is already the case on mobile, since `dau == qdau`, even though qualifying conditions exist (e.g. we don't count client_ids coming from BrowserStack).

I apologize for not communicating this sooner; the definitions were still undecided when this work was scoped. This isn't blocking for the current PR since I think it's more important to get the pipeline into place.

Reply:
Sure thing.
In Desktop's view `firefox_desktop.active_users_aggregates` we have DAU and QDAU too because the file is auto-generated using a template. If we want to change it we can do that in a separate PR.